tools/ci/tc/decision.py

```diff
@@ -238,7 +238,14 @@ def create_tc_task(event, task, taskgroup_id, depends_on_ids, env_extra=None):
             "env": task.get("env", {}),
         },
         "extra": {
-            "github_event": json.dumps(event)
+            "github_event": json.dumps(event),
+            "github": {
```

Comment:
Could we make this part of the yml file rather than hardcoded here?

Reply:
Oh absolutely, I'm just playing around atm :)